.github/workflows/delete_lambda_layer.yml

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2023 SolarWinds, LLC.
+# All rights reserved.
+
+name: Build and publish ruby lambda layer
+
+on:
+  workflow_dispatch:
+    inputs:
+      layer-version:
+        required: true
+        description: 'Layer version that need to be deleted'
+
+permissions:
+  id-token: write   # This is required for requesting the JWT
+  contents: read    # This is required for actions/checkout
+
+jobs:
+  delete_layer:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        aws_region: 
+          - ap-northeast-1
+          - ap-northeast-2
+          - ap-south-1
+          - ap-southeast-1
+          - ap-southeast-2
+          - ca-central-1
+          - eu-central-1
+          - eu-north-1
+          - eu-west-1
+          - eu-west-2
+          - eu-west-3
+          - sa-east-1
+          - us-east-1
+          - us-east-2
+          - us-west-1
+          - us-west-2
+
+    steps:
+      - uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: <arn:aws:iam::111122223333:role/GitHubAction-AssumeRoleWithAction>
```

Comment:
I am thinking to remove the workflow since I found out we can manually remove layer from aws console, but I am not sure if we have the ability/permission to do in production as well

Reply:
Agree, can remove this workflow and delete staging layers via the console or `aws` CLI. It's very unlikely that we'd go to the extreme of deleting a production layer.